### ISSUE TITLE
Dialog Full Screen component reports 5 accessibility violations: 

### ISSUE THREAD

**Issue by DianeGit22** [Sage/carbon#4793]
### Current behaviour

Dialog Full Screen component "with complex example" story reports 5 accessibility violations (carbon.sage.com): 
![image](https://user-images.githubusercontent.com/47564230/152576054-2936534b-33de-4856-a3e3-2ed083f2099f.png)

### Expected behaviour

No accessibility violations should occur. 

### CodeSandbox or Storybook URL

https://carbon.sage.com/?path=/story/dialog-full-screen--with-complex-example

### Carbon Version

Latest

### What browsers are you seeing the problem on?

Firefox, Chrome, Microsoft Edge

### What Operating System are you seeing the problem on?

Windows

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by nicktitchmarsh** [comment]
Looks like partly a duplicate of #4692. I will add these issues to that ticket

**Comment by nicktitchmarsh** [comment]
FE-4922
